pytorch_lightning/core/memory.py

```diff
@@ -13,14 +13,8 @@
 # limitations under the License.
 from pytorch_lightning.utilities import rank_zero_deprecation
 
-rank_zero_deprecation(
-    "`pytorch_lightning.core.memory.get_memory_profile` and"
-    " `pytorch_lightning.core.memory.get_gpu_memory_map` have been moved"
-    " to `pytorch_lightning.utilities.memory` since v1.5 and will be removed in v1.7."
-)
-
 # To support backward compatibility as get_memory_profile and get_gpu_memory_map have been moved
-from pytorch_lightning.utilities.memory import get_gpu_memory_map, get_memory_profile  # noqa: E402, F401 # isort: skip
+from pytorch_lightning.utilities.memory import get_gpu_memory_map  # noqa: F401 # isort: skip
```

Comment:
I think you should remove this too, because you removed the test and the combined deprecation warning above

Reply:
@puhuk pls ^^